ledger-core/virtual/validation/transcript.go

```diff
@@ -0,0 +1,15 @@
+package validation
+
+type Transcript struct {
+	Entries []TranscriptEntry
```

Comment:
Considering, that Transcript will be big (~1M), I strongly recommend 
* prefer to use by-value / no pointer / no slice types
* make a paged storage from the very beginning (see lineage.lineRecords for a copy-paste implementation)
* use integer indices